cypress/integration/admin_portal/pages/enterprise_code_management.js

```diff
@@ -0,0 +1,33 @@
+class CodeManagementPage {
+  getFormField(field_name) {
+    return cy.get(`[name="${field_name}"]`)
+  }
+
+  getLabels(label_name) {
+    const labels = {'Email': 1, 'Company': 2, 'Number of Codes':3}
+    return cy.get(`.form-group:nth-of-type(${labels[label_name]}) label`)
+  }
+
+  getRequestCodesButton(){
+    return cy.get('[class="btn btn-primary"]')
```

Comment:
All those years we have been using classes simply with dot notation, why have you suddenly switched to the above expression